esmvaltool/_data_finder.py

```diff
@@ -119,15 +123,27 @@ def get_input_dirname_template(variable, rootpath, drs):
     _drs = drs.get(project, 'default')
     input_dir = cfg['input_dir']
     if isinstance(input_dir, six.string_types):
-        dir2 = replace_tags(input_dir, variable)
+        dir2 = replace_tags(input_dir, variable, 0)
     elif _drs in input_dir:
-        dir2 = replace_tags(input_dir[_drs], variable)
+        try:
```

Comment:
Remove the `try` ... `except` block, not finding the institute matching a model is an error in the `config-developer.yml` file and should not be silently ignored.

Reply:
you need the try/except bit since if it's list of models for a certain institution will have a variable, another institution will have a different variable and ```variable['model']``` will spit out a ```KeyError```, it's not that nice but maybe add an else statement?